### PR TITLE
Show tips to write great questions

### PR DESCRIPTION
**Pull request by Errorname**
**Preview of the tips :**

image

### CODE REVIEW

File: src/scenes/Question/scenes/Edit/components/Tips/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  render () {
+    return (
+      <Card {...this.props}>
+        <CardText>
+          <h3 style={{ textAlign: 'center' }}>
+            Tips to write great questions on the FAQ
+          </h3>
+          <List selectable>
+            {tips.good.map((tip, i) => (
+              <ListItem
```
Comment (frinyvonnick) [inline src/scenes/Question/scenes/Edit/components/Tips/index.js]: Is it possible to make a reusable component ? There seems to be some duplicated code...

Comment (Errorname) [reply src/scenes/Question/scenes/Edit/components/Tips/index.js]: Are you talking about the maps ?

Comment (frinyvonnick) [reply src/scenes/Question/scenes/Edit/components/Tips/index.js]: The ListItem

Comment (Errorname) [reply src/scenes/Question/scenes/Edit/components/Tips/index.js]: Done 👌

**Pull request merged**